src/pytorch_lightning/tuner/lr_finder.py

```diff
@@ -95,16 +96,16 @@ def __init__(self, mode: str, lr_min: float, lr_max: float, num_training: int):
         self.lr_max = lr_max
         self.num_training = num_training
 
-        self.results = {}
+        self.results = {}  # type: Dict[str, Any]
```

Comment:
Shall we use the following syntax for consistency in the codebase? (kind of related to [PEP526](https://peps.python.org/pep-0526/))
```suggestion
        self.results: Dict[str, Any] = {}
```